bigtable/admin.go

```diff
@@ -294,6 +313,17 @@ func (ac *AdminClient) CreateTableFromConf(ctx context.Context, conf *TableConf)
 		tbl.ChangeStreamConfig = &btapb.ChangeStreamConfig{}
 		tbl.ChangeStreamConfig.RetentionPeriod = durationpb.New(conf.ChangeStreamRetention.(time.Duration))
 	}
+	if conf.AutomatedBackupPolicy != nil && *conf.AutomatedBackupPolicy != DisableAutomatedBackupPolicy {
```

Comment:
Having 2 ways to describe the same thing is not that great :(

Reply:
Thanks. Agreed. Updated (Discussed offline)